domains/runtime/core-evm/src/runtime.rs

```diff
@@ -161,7 +165,9 @@ impl_opaque_keys! {
 
 #[sp_version::runtime_version]
 pub const VERSION: RuntimeVersion = RuntimeVersion {
-    spec_name: create_runtime_str!("subspace-evm-domain"),
+    // TODO: change this back to `subspace-evm-domain` once the following PR is merged
+    // https://github.com/polkadot-js/apps/pull/9399
```

Comment:
This PR has been merged.

Reply:
yes. I just noticed. but its not yet deployed. We are still testing :)